### DISCUSSION TITLE
Does EditorElement currently work?

### DISCUSSION THREAD

**Discussion by pbrink21** [vueform/vueform#273] [Questions] [answered]
The short version is this: 
I was using EditorElement in my forms. I recently updated VueForm because there was a bugfix I needed (I don't remember what...), but this has seemed to break all my EditorElements.
I've only tried it in a couple of places, but here's the skinny:
my form looks something like this:
```
<script>
  export default { 
    methods: {
      submit (data) {
        console.log(data)
        *send axios request to back end*
      }
    }
  }
</script>

<template>
  <Vueform :display-errors="false" :endpoint="false" @submit="async function(form$){ submit(form$.requestData) }" ref="form$">
    //... other form stuff ...
    <EditorElement name="description" label="Description" placeholder="description..."/>
  </Vueform>
</template>
```
The `console.log` in the `submit` function shows the `description` property coming back as null. And doing `this.$refs.form$.el$('description').value` also returns null
This seems to be the case for every form that I have... this kinda leads me to believe it's a bug within `EditorElement` itself, but it could also presumably be something within my configuration. Any reccomendations?

**Comment by pbrink21** [comment]
My version of `vue` is `3.4.30`. My version of `vueform` is `1.9.12`

**Reply by adamberecz** [reply, marked as answer]
Having this put into a sandbox there seems to be no issue: https://stackblitz.com/edit/github-dhvmkd?file=src%2FApp.vue

There were updates targeting the EditorElement in the last few patches though.

I'd recommend package-lock and node_modules removal and a complete npm reinstall to see if that solves it.

**Reply by pbrink21** [reply]
Wow, funky - that's not something I've had to do before. But that fixed it! Good to know, thank you so much for all the help!
